### ISSUE TITLE
Portraits page is blank for all characters still

### ISSUE THREAD

**Issue by Bob423** [BarleyFlour/VisualAdjustments2#55]
I checked to see if this had been talked about and the issue was closed, but in the current version, this is still a problem for me.

**Comment by Bob423** [comment]
Important note I just realized: This only seems to be an issue for my brand new save file that's before Kenabres

**Comment by bmardiney** [comment]
Yeah this is totally broken in my game. Can't change my black horse's portrait (or anyone else's).
